### ISSUE TITLE
Arrays equals

### ISSUE THREAD

**Issue by elect86** [kotlintest/kotlintest#116]
Is it possible to have an option to compares arrays?

[Related](http://stackoverflow.com/questions/40691398/kotlin-how-to-test-intarray)

**Comment by sksamuel** [comment]
I've added this to release 2.0

**Comment by helmbold** [comment]
Why is this closed? I can not see another issue for it.

**Comment by sksamuel** [comment]
I've pushed code for it.

**Comment by helmbold** [comment]
Ah, I did not see that :-) Don't forget to mention it in the readme.md.

**Comment by elect86** [comment]
Is it relatively stable the 2.0? Shall I give it a try?

**Comment by sksamuel** [comment]
Yeah should be. We just might change the matchers a bit before release.

**Comment by elect86** [comment]
Cool!

Ps: I tried to modify 

`    testCompile("io.kotlintest:kotlintest:1.3.5")`

to

`    testCompile("io.kotlintest:kotlintest:release/2.0-SNAPSHOT")`

but it doesn't resolve it.. and neither 

`    testCompile("io.kotlintest:kotlintest:2.0-SNAPSHOT")`

I am using Jitpack, do you maybe have a clue why?

**Comment by sksamuel** [comment]
There's no published artifact, you'd have to build from source.

**Comment by elect86** [comment]
In theory it can do it.. I'll see with them what is wrong

**Comment by kevinherron** [comment]
Will this make it into a 1.3.x release or is it only going into 2.0? Any guess on a 2.0 release date?

**Comment by sksamuel** [comment]
The 2.0 is targeted with Kotlin 1.1, so really anytime soon. I think the only thing left is for us to me to finish a bit of work on matchers @helmbold ?

**Comment by helmbold** [comment]
@sksamuel There are some more [things to do](https://github.com/kotlintest/kotlintest/milestone/4), before we can release 2.0. I'm looking for a way to bring `config` to behavior spec, but my builder approach doesn't work as expected so far.
